### PR TITLE
Allow front end functionality that sets player's equipped accessories by adding doPost method

### PR DESCRIPTION
**Pull request by lzoebaker**
This doPost method allows Leah's front-end profile page (to be added in the next PR) to set the player's equipped accessories.

### CODE REVIEW

File: src/test/java/com/google/sps/GetEquippedAccessoriesTest.java [hunk 1]
```diff
@@ -221,6 +224,138 @@ public void getEquippedAccessoriesWithLoggedOutUser_ExceptionMessageWritten() th
...
+        user, accessories, equippedHatID, equippedGlassesID, equippedCompanionID);
+
+    // add new accessories that will be equipped to database
+    createAccessoryAndAddToDatabase(COWBOY_HAT_ID, Type.HAT);
+    createAccessoryAndAddToDatabase(SUNGLASSES_ID, Type.GLASSES);
+    createAccessoryAndAddToDatabase(CAT_ID, Type.COMPANION);
+
+    // post request changing player accessories, first two values are not valid accessory ids
+    String equippedHatInput = null;
+    String equippedGlassesInput = "garbage";
```
Comment (lattai) [inline src/test/java/com/google/sps/GetEquippedAccessoriesTest.java:324]: Make this a variable

Comment (menynexi) [reply src/test/java/com/google/sps/GetEquippedAccessoriesTest.java:324]: +1

File: src/main/java/com/google/sps/servlets/GetEquippedAccessories.java [hunk 2]
```diff
@@ -88,4 +88,40 @@ private JsonElement getJsonTreeFromID(String id) {
...
+      response.setContentType(HTML_CONTENT_TYPE);
+      response.getWriter().println(LOGGED_OUT_EXCEPTION);
+    }
+  }
+
+  /* Safeguard function to ensure that a player always has either a valid accessory id set or
+   * the NONE_EQUIPPED string (which is used in this servlets doGet method to determine whether
+   * or not to send an AccessoryObject or just a NONE_EQUIPPED message.
+   */
+  private String defaultIfNotFound(String id) {
```
Comment (menynexi) [inline src/main/java/com/google/sps/servlets/GetEquippedAccessories.java:113]: maybe reword id to be accessoryId
